test/core-test/runtime-field.test.js

```diff
@@ -0,0 +1,46 @@
+import test from 'ava';
+import RuntimeField from '../../src/core/runtime-field';
+
+test('constructor set arguments', t => {
+    const valueA = new RuntimeField(
+        'keyword',
+        "emit(doc['name'].value)"
+    ).toJSON();
+    const valueB = new RuntimeField(
+        'keyword',
+        "emit(doc['name'].value)"
+    ).toJSON();
+    t.deepEqual(valueA, valueB);
```

Comment:
What are we trying to test here? Was the 2nd instance meant to be populated via method calls?